framework/widgets/ActiveField.php

```diff
@@ -829,7 +815,6 @@ protected function getClientOptions()
         if ($clientValidation) {
             $validators = [];
             foreach ($this->model->getActiveValidators($attribute) as $validator) {
-                /* @var $validator \yii\validators\Validator */
```

Comment:
instead it move up

Reply:
Why should I use it? That can get a Validator.
![image](https://user-images.githubusercontent.com/19358691/135744140-c4801267-df9a-4f58-9ada-1ee0ea1d167c.png)

Reply:
@toir427 1. not all use phpstorm 2. code inspections work better with annotations

Reply:
ok